src/data_scripts/column_cofig.py

```diff
@@ -0,0 +1,16 @@
+"""
+dict
+key: what sort of encoding needed to be performed
+values: columns that need to be encoded base of their key(encoding)
+"""
+COLS_CONFIG = {
```

Comment:
Why did you use a dictionary here instead of other data structures?

Reply:
Using a dictionary for COLS_CONFIG was a strategic choice to manage the encoding of categorical features effectively. The dictionary structure allows for a clear and intuitive mapping of encoding types to the corresponding columns that require such processing. For example, one_hot_encode_cols is a key that indicates which columns should undergo one-hot encoding. When we need to perform the encoding, the process is straightforward: we call the function one_hot_encode_cols and pass the relevant columns by referencing the dictionary key like so one_hot_encode_cols(COLS_CONFIG["one_hot_encode_cols"]). This will apply one-hot encoding to the "Type" and "Gender" columns.